### ISSUE TITLE
ZSH on WSL2 job table full or recursion limit exceeded

### ISSUE THREAD

**Issue by skhaz** [microsoft/vscode#165648]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.73.0
- OS Version: Windows 11 WSL2 Ubuntu

Steps to Reproduce:

1. Open any project
2. See the error on console

```
/tmp/ubuntu-code-zsh/.zshenv:9: job table full or recursion limit exceeded
Panzer% zsh
/tmp/ubuntu-code-zsh/.zshenv:9: job table full or recursion limit exceeded
Panzer% cat /tmp/ubuntu-code-zsh/.zshenv
# ---------------------------------------------------------------------------------------------
#   Copyright (c) Microsoft Corporation. All rights reserved.
#   Licensed under the MIT License. See License.txt in the project root for license information.
# ---------------------------------------------------------------------------------------------
if [[ -f $USER_ZDOTDIR/.zshenv ]]; then
        VSCODE_ZDOTDIR=$ZDOTDIR
        ZDOTDIR=$USER_ZDOTDIR

        . $USER_ZDOTDIR/.zshenv

        USER_ZDOTDIR=$ZDOTDIR
        ZDOTDIR=$VSCODE_ZDOTDIR
fi
```

**Comment by Navid61** [comment]
I've same problem but my os is Debian 11 x64

**Comment by deejinng** [comment]
What could be the solution pls

![IMG_20221113_121357_648~2](https://user-images.githubusercontent.com/107402108/201518839-7f52ac2a-c5dc-4f9a-9c8a-f7bf9b316a7f.jpg)

Lol

**Comment by Tyriar** [comment]
@deejinng it's fixed in https://github.com/microsoft/vscode/issues/163843, you could make the change to the scripts in VS Code's install dir manually to workaround the issue:

https://github.com/microsoft/vscode/pull/165174/files

**Comment by deejinng** [comment]
Thanks @Tyriar 
I restarted VS Code and it worked fine.

**Comment by shateelahmed** [comment]
> @deejinng it's fixed in #163843, you could make the change to the scripts in VS Code's install dir manually to workaround the issue:
> 
> https://github.com/microsoft/vscode/pull/165174/files

@Tyriar What is the location of these VS Code scripts?

**Comment by muhammadsiddiqbe** [comment]
> Thanks @Tyriar I restarted VS Code and it worked fine.

😆 worked thanks. It is interesting it was not working in another terminal windows outside of vscode.
